### DISCUSSION TITLE
Import from Tellico

### DISCUSSION THREAD

**Discussion by KVAnton-WEB** [mateusz-bak/openreads#705] [Ideas]
**Is your feature request related to a problem? Please describe.**
I can't import from Tellico. When importing CSV the application either crashes or imports empty records (Without filling fields) 

**Describe the solution you'd like**
Import from Tellico, preferably along with the covers. In Tellico, the native format is in zip with xml and picture directory, but supports other export methods as well.

And probably need to add a description of the supported csv format or add an example of such a file, (then you can just try to convert, which may be useful to many, because do not know how many different programs can be)

**Additional context**
[Tellico is an application for organizing your collections (KDE Applications)](https://apps.kde.org/tellico/)

**Comment by mateusz-bak** [comment]
@KVAnton-WEB  can you share an example CSV from Tellico?

**Comment by astrorobby** [comment]
Here's an XSLT file that works with Tellico to export to CSV in the format described in [csv.md](https://github.com/mateusz-bak/openreads/blob/master/doc/csv.md)

Rename it to tellico2openreads.xsl, then use `File->Export XSL Transform` within Tellico to use it...
[tellico2openreads.txt](https://github.com/user-attachments/files/17671945/tellico2openreads.txt)
